services/api/src/messaging/players.service.ts

```diff
@@ -32,6 +32,23 @@ const findPlayerById = (
   );
 };
 
+const checkIfGameComplete = (
```

Comment:
Please rename this method, to better represent the value that is returned. Currently it suggests that returned value is `boolean`.